thirdpartynotices.html

```diff
@@ -598,4 +694,420 @@
  (D) If you distribute any portion of the software, you must retain all copyright, patent, trademark, and attribution notices that are present in the software.
  (E) If you distribute any portion of the software in source code form, you may do so only under this license by including a complete copy of this license with your distribution. If you distribute any portion of the software in compiled or object code form, you may only do so under a license that complies with this license.
  (F) The software is licensed &quot;as-is.&quot; You bear the risk of using it. The contributors give no express warranties, guarantees or conditions. You may have additional consumer rights under your local laws which this license cannot change. To the extent permitted under your local laws, the contributors exclude the implied warranties of merchantability, fitness for a particular purpose and non-infringement.
-</pre></details></li></ol></body></html>
\ No newline at end of file
+</pre></details></li><li><details><summary>Microsoft.TeamFoundation.DistributedTask.Common.Contracts 16.153.0 - OTHER</summary><ul><li>(c) 2008 VeriSign, Inc.</li><li>(c) Microsoft Corporation.</li></ul><pre>MICROSOFT SOFTWARE LICENSE TERMS
+
+MICROSOFT .NET LIBRARY
+
+These license terms are an agreement between Microsoft Corporation (or based on where you live, one of its affiliates) and you. They apply to the software named above. The terms also apply to any Microsoft services or updates for the software, except to the extent those have different terms.
+
+IF YOU COMPLY WITH THESE LICENSE TERMS, YOU HAVE THE RIGHTS BELOW.
+
+1.    INSTALLATION AND USE RIGHTS.
+You may install and use any number of copies of the software to design, develop and test you�re applications.  You may modify, copy, distribute or deploy any .js files contained in the software as part of your applications.
```

Comment:
These goofy characters shouldn't be here, and they show up in file. I'll see what I can do to address them

Reply:
Heard back from legal. The notice generation tool is harvesting the data correctly, but the data contains extra character sequences. We're supposed to use the unmodified license files, so we inherit the noise.